packages/protobuf-encoder/generated/message_pb.d.ts

```diff
@@ -0,0 +1,148 @@
+import $protobuf from "protobufjs";
```

Comment:
For some reason I had to change this import manually after compilation, it was generated as `import * as $protobuf from "protobufjs"`, and that is not correct. I am going to ask about it in protobufjs repo issues

Reply:
I filed an issue https://github.com/protobufjs/protobuf.js/issues/1657